core/db/models.py

```diff
@@ -32,10 +34,27 @@ class Meta:
 class ConnectionPeerModel(BaseModel):
     user = ForeignKeyField(UserModel, backref="peer", on_delete="CASCADE")
     public_key = CharField()
+    private_key = CharField()
     preshared_key = CharField()
     shared_ips = CharField()
     peer_name = CharField(default=None, null=True)
 
+    def setup_peer_keys(self):
+        peer_private_key = private_key()
+        self.privatekey = peer_private_key
+        self.public_key = public_key(peer_private_key)
+        self.preshared_key = preshared_key()
```

Comment:
why is it here? this is not a Model method. setup methods should be in `config.settings` module.

P.S. it's already in `config.settings`